src/patternfly/components/Chips/chip-group.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../patternfly-utilities";
+
+.pf-c-chip-group {
+  --pf-c-chip-group--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-chip-group--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-chip-group--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-chip-group--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--MarginBottom: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--m-toolbar--BackgroundColor: var(--pf-global--BackgroundColor--300);
+  --pf-c-chip-group--m-toolbar-MaxWidth: #{pf-size-prem(500px)}; // for demo
+  --pf-c-chip-group--m-select-BackgroundColor: transparent;
+  --pf-c-chip-group--m-select-Width: #{pf-size-prem(375px)}; // for demo
```

Comment:
`--pf-c-chip-group--m-select-PaddingBottom: var(--pf-global--spacer--xs);` would give select exact padding w/addition of magic value to `pf-c-chip`.